### ISSUE TITLE
Request for 1.6.0 package for Windows

### ISSUE THREAD

**Issue by ajraymond** [zdia/gorilla#198]
I've had a request by email from someone who is using the beta 1.6.0 version I packaged for Ubuntu, and would like to run the same version on Windows.

Are there plans to update the OSX and Windows packages to 1.6.0?

**Comment by rich123** [comment]
Yes, that's been "a plan" for some time now.  I just seem to keep getting distracted by other "real life" items such that it never bubbles up to the top of my priority list.

Maybe this is as good a time as any to go ahead and wrap up a 1.6.0 beta for windows (and Mac) using the same commit as the Ubuntu version and put them up for download.

**Comment by rich123** [comment]
Ok, you gave me a little 'kick' that I needed to go ahead and package up something to release.  There is now a 1.6.0-beta-1 .kit file on the downloads page (https://gorilla.dp100.com/downloads/).  It (the .kit) is OS independent, so a TclKit executable from the support page (https://gorilla.dp100.com/downloads/tclkit/) will be needed to launch it and test.

The Ubuntu user who emailed you can use the .kit, plus one of the MS Windows TclKit executable files, to run 1.6.0 on Windows.

Also, I don't know exactly which commit you packaged for Debian/Ubuntu (thanks very much for the help in creating the Debian/Ubuntu packages), but the new 1.6.0-beta-1 contains commit  99f0e269cf4f6f07d755f54b981a3c09ead00b90 which fixed a very longstanding bug with handling of UTF-8 data strings.  If your Debian/Ubuntu package is from before this commit, I do recommend updating.  1.6.0-beta-1 also contains updated Italian translation files by Marco Ciampa (thanks to Marco for that assistance), which might be important to any Italian users of Gorilla.

**Comment by rich123** [comment]
A new 1.6.0-beta-2 .kit package has been released.
